scripts/notebook_entrypoint.sh

```diff
@@ -31,6 +31,12 @@ elif [ "$JUPYTER_MODE" = "jupyterhub" ]; then
   unset NOTEBOOK_DIR
 
   jupyterhub -f "$JUPYTERHUB_CONFIG_DIR"/jupyterhub_config.py
+elif [ "$JUPYTER_MODE" = "jupyterhub-singleuser" ]; then
+  echo "Starting Jupyter Notebook for user: $JUPYTERHUB_USER"
+  unset NOTEBOOK_DIR
+
+  python /src/jupyterhub_config/hub_singleuser.py
+
```

Comment:
There's going to be a lot of jupyterlab and jupyterhub multiuser cruft to clean up at some point...